src/pyhf/infer/mle.py

```diff
@@ -132,7 +132,7 @@ def fixed_poi_fit(
     This is done by minimizing the objective function of :func:`~pyhf.infer.mle.twice_nll`
     of the model parameters given the observed data, for a given fixed value of :math:`\mu`.
     This is used to produce the constrained maximal likelihood for the given :math:`\mu`
-    :math:`L\left(\mu, \hat{\hat{\boldsymbol{\theta}}}\right)` in the profile
+    ,:math:`L\left(\mu, \hat{\hat{\boldsymbol{\theta}}}\right)`, in the profile
```

Comment:
When I build the docs locally the first comma that has no space between it and the `:math:` breaks Sphinx syntax, and if I then put a space `, :math:` the render looks a bit strange. Can you remove the first comma from line `135` 
```suggestion
    :math:`L\left(\mu, \hat{\hat{\boldsymbol{\theta}}}\right)`, in the profile
```
and put it at the end of line `134` instead (just after the :math:`\mu`)?